cstore_fdw.h

```diff
@@ -259,6 +271,8 @@ typedef struct TableWriteState
 	StripeData *stripeData;
 	StripeSkipList *stripeSkipList;
 	uint32 stripeMaxRowCount;
+	BlockDataBuffers **blockDataBuffersArray;
+	StringInfo decompressionBuffer;
```

Comment:
Can we put a comment before this field to explain what is the reason for having ```decompressionBuffer```?